subprojects/docs/src/docs/userguide/more_about_tasks.adoc

```diff
@@ -481,169 +582,142 @@ For this example, we’re using a custom class to represent the model data. Howe
 +
 The directory where the generated files go. As with input files, there are several annotations for output files and directories. A property representing a single directory requires `@OutputDirectory`. You’ll learn about the others soon.
 
-These annotated properties mean that Gradle will skip the task if none of the source files, template engine, model data or generated files have changed since the previous time Gradle executed the task. This will often save a significant amount of time. You can learn how Gradle detects <<sec:how_does_it_work,changes later>>.
+These annotated properties mean that Gradle will skip the task if none of the source files, template engine, model data or generated files have changed since the previous time Gradle executed the task. This will often save a significant amount of time. You can learn how Gradle detects <<#sec:how_does_it_work,changes later>>.
 
-This example is particularly interesting because it works with collections of source files. What happens if only one source file changes? Does the task process all the source files again or just the modified one? That depends on the task implementation. If the latter, then the task itself is incremental, but that’s a different feature to the one we’re discussing here. Gradle does help task implementers with this via its <<incremental_tasks,incremental task inputs>> feature.
+This example is particularly interesting because it works with collections of source files. What happens if only one source file changes? Does the task process all the source files again or just the modified one? That depends on the task implementation. If the latter, then the task itself is incremental, but that’s a different feature to the one we’re discussing here. Gradle does help task implementers with this via its <<custom_tasks.adoc#incremental_tasks,incremental task inputs>> feature.
 
 Now that you have seen some of the input and output annotations in practice, let’s take a look at all the annotations available to you and when you should use them. The table below lists the available annotations and the corresponding property type you can use with each one.
 
-.Incremental build property type annotations
-[cols="a,a,a", options="header"]
-|===
-| Annotation
-| Expected property type
-| Description
 
-| `@api:org.gradle.api.tasks.Input[]`
-| Any serializable type
-| A simple input value
+===== Incremental build property type annotations
```

Comment:
I think this is a step backwards in terms of readability and discoverability from the table representation.  I think you can see that because we're trying to add context around the expected property types. I think a table was the right presentation for this, even if it looks poor on some devices.

Reply:
Okay, I'll change it back